src/dos/dos_programs.cpp

```diff
@@ -1179,7 +1179,11 @@ class INTRO : public Program {
 		/* Only run if called from the first shell (Xcom TFTD runs any intro file in the path) */
 		if (DOS_PSP(dos.psp()).GetParent() != DOS_PSP(DOS_PSP(dos.psp()).GetParent()).GetParent()) return;
 		if (cmd->FindExist("cdrom",false)) {
-			WriteOut(MSG_Get("PROGRAM_INTRO_CDROM"));
+#if (WIN32)
```

Comment:
This should be either
```C
#ifdef WIN32
```
or
```C
#if defined(WIN32)
```

Reply:
Thanks!  I thought it was unusual to see it handled this way too, but I was keeping with the convention used there.  I made a commit to address that.